practice-app/backend/api/likes/likes.py

```diff
@@ -1,48 +1,87 @@
-from flask import Blueprint, jsonify
+from flask import Blueprint, jsonify, abort
 from database import mongo
 import datetime
 
-likes_bp = Blueprint('Likes of post', __name__)
+likes_bp = Blueprint('Like action and viewing user which likes post', __name__)
 
 @likes_bp.route('/api/post/<int:postId>/likes', methods=['GET'])
-
 def getLikes(postId):
+    postExist = findPostFromDb(postId)
+    userThatLikesPost = getLikesFromDb(postId)
+    numberOfPostLike = list(userThatLikesPost)
```

Comment:
I could not see where you use this variable later in the function, it may be unnecessary

Reply:
It was unnecessary, so I forgot to reply in here. Thanks